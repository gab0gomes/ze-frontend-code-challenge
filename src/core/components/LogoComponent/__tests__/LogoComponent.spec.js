import { mount, createLocalVue } from '@vue/test-utils'
import VueRouter from 'vue-router'

import Logo from '../index'

const localVue = createLocalVue()
localVue.use(VueRouter)

const routes = [
  {
    path: '/',
    name: 'Home',
    component: { template: '<div>foo</div>' }
  }
]

const router = new VueRouter({
  routes,
  linkActiveClass: 'active',
  mode: 'history'
})

describe('LogoComponent', () => {
  it('render correctly', async () => {
    const wrapper = mount(Logo, {
      localVue,
      router
    })

    expect(wrapper.element).toMatchSnapshot()

    wrapper.destroy()
  })
})
