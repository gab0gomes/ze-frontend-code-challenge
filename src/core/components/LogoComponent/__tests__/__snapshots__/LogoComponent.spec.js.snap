// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoComponent render correctly 1`] = `
<img
  alt="Logotipo do Zé Delivery, possui uma ilustração de um rosto masculino a esqueda em preto com fundo amarelo e a direita está escrito Zé em branco"
  class="router-link-exact-active active"
  src="../static/images/logo.png"
/>
`;
